ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/views/PaywallBackground.kt

```diff
@@ -0,0 +1,32 @@
+package com.revenuecat.purchases.ui.revenuecatui.views;
```

Comment:
Instead of `views`, I would suggest usinga `composables` package. Thoughts?

Reply:
Good point, moved.